ui/app/styles/helper-classes/flexbox-and-grid.scss

```diff
@@ -104,6 +104,28 @@
   flex: 50%;
 }
 
+// moving away from !important, fresh flex styles below
+.flex {
+  display: flex;
+  // direction
+  &.row-wrap {
+    flex-flow: row wrap;
+  }
+
+  // alignment
+  &.space-between {
+    justify-content: space-between;
+  }
+
+  &.row-gap-8px {
```

Comment:
I can also these to `row-gap-m` and we can move to numerical sizing when we've generally cleaned up the CSS to use that more consistently

Reply:
If we're changing to numerical styles, it might be good to with `.row-gap-8`. I noticed in most [bootstrap helper classes](https://getbootstrap.com/docs/4.0/utilities/spacing/#examples) that it's formatted with the number at the end.